src/atoms/LinkBarLinkBordered.js

```diff
@@ -0,0 +1,43 @@
+import styled from 'styled-components';
+import { LinkBarLink } from '..';
+import { stringToRGB } from '../utils';
+
+const calculateTextColor = (backgroundColor, theme) => {
+	const { r, g, b } = stringToRGB(backgroundColor);
+	const yiq = ((r * 299) + (g * 587) + (b * 114)) / 1000; // converting to YIQ to determine brightness of background
+	return (yiq >= 150) ? theme.colors.typeGray : theme.colors.white;
+};
+
+const LinkBarLinkBordered = styled(LinkBarLink)`
+	border-radius: 0.3rem;
+	border: solid 0.1rem ${props => props.theme.colors.borderGray};	
+	color: ${props => props.theme.colors.typeGray} !important;
+	margin: 0 0.5rem 0.5rem;
```

Comment:
We should use theme variables here.

All other LinkBar* components share the same combined height, so that when a LinkBarButton and a LinkBarLink are next to each other on a HorizontalLinkBar, both will take up the same amount of vertical space. This component should follow suit, making sure that line-height + padding + margin is the same height as a LinkBarLink.

Reply:
... so if we add vertical margin, we should decrease padding or line-height.

Also, if the vertical margins were symmetrical, it would look better in a colored HorizontalLinkBar.